libs/auth-nest-tools/src/lib/current-user.decorator.ts

```diff
@@ -3,37 +3,25 @@ import {
   ExecutionContext,
   UnauthorizedException,
 } from '@nestjs/common'
-import { GqlExecutionContext } from '@nestjs/graphql'
 
 import { User } from './user'
-
-export const CurrentUser = createParamDecorator(
-  (data: unknown, context: ExecutionContext): User => {
-    const ctx = GqlExecutionContext.create(context)
-    const request = ctx.getContext().req
-    const user = request.user
-
-    return {
-      ...user,
-      authorization: request.headers.authorization,
-    }
-  },
-)
+import { getRequest } from './getRequest'
 
 export const getCurrentUser = (context: ExecutionContext): User => {
-  const request = context.switchToHttp().getRequest()
-  const user = request.user
+  const request = getRequest(context)
 
+  const user = request.user
   if (!user) {
-    throw new UnauthorizedException('You are not authenticated')
-  }
-
-  return {
-    ...user,
-    authorization: request.headers.authorization,
+    console.warn(
```

Comment:
Is logging to console done on purpose?

Reply:
Nope! Should be using the common logger. Good catch!